tools/migration/main.go

```diff
@@ -78,17 +78,39 @@ func main() { // nolint: deadcode
 	switch command {
 	case "-h", "--help":
 		showUsageAndExit(0)
-	case "describe", "buildonly", "migrate", "install":
-		logFile, err := openLogFile()
+	case "describe":
```

Comment:
Even though there is some duplicated code here, I separated this out because it doesn't need to do nearly as much as the others.